eng/pipelines/ci.yml

```diff
@@ -13,3 +13,8 @@ jobs:
 
       - script: npm run test
         displayName: Run Tests
+
+      - script: |
+          npm run coverage-push -- $(Build.Repository.Name) $(Build.SourceBranch) $(github-token) $(storage-coverage-user) $(storage-coverage-pass)
```

Comment:
In package.json, we don't have `coverage-push`, do we need to add that before this CI change work?

Reply:
No, it's provided by `autorest.typescript`, you can see the script step has a `workingDirectory` set to that folder path so the `npm` script is coming from there instead.

Reply:
In  https://github.com/Azure/autorest.swift/blob/master/AutorestSwiftTest/ZZZAutoRestReportTest.swift,  we print out the coverage values like:

is this the right format that the coverage site is expecting?


```
        let coverage: Float = Float(passedCount) / Float(totalCount) * 100

        print("Passed Test=\(passedCount), Total Test=\(totalCount), Coverage=\(String(format: "%.2f", coverage))%")
```